apps/webapp/app/components/primitives/Switch.tsx

```diff
@@ -15,7 +15,7 @@ const variations = {
     container: "flex items-center gap-x-1.5 rounded hover:bg-slate-850 pr-1 py-[0.1rem] pl-1.5",
     root: "h-3 w-6",
     thumb: "h-2.5 w-2.5 data-[state=checked]:translate-x-2.5 data-[state=unchecked]:translate-x-0",
-    text: "text-xs text-slate-400 group-hover:text-slate-200 mt-0.5",
```

Comment:
@hmacr could you provide a screen grab of the toggle switch on the Integration page to check this margin change hasn't altered the layout of that Component?

Reply:
@samejr Here's the screen grab showing that both the toggle switches are center aligned vertically.

https://github.com/triggerdotdev/trigger.dev/assets/132386067/89e95194-3ae1-4af4-b1ba-72478db958bd